src/pages/Dashboard/Actions/AddActionCard.vue

```diff
@@ -34,11 +35,29 @@ export default {
         select: {
           title: '...'
         }
-      }
+      },
+      SLATypes: [
+        { name: 'starts but does not finish', enum: 'STARTED_NOT_FINISHED' },
+        {
+          name: 'is scheduled but does not start',
```

Comment:
Note that the check duration begins at the scheduled start time not the flow 'Scheduled' state transition time.